src/main/java/com/google/devtools/build/lib/rules/android/AndroidConfiguration.java

```diff
@@ -230,7 +230,9 @@ public static class Options extends FragmentOptions {
           OptionEffectTag.LOADING_AND_ANALYSIS,
           OptionEffectTag.LOSES_INCREMENTAL_STATE,
         },
-        help = "The Android target CPU.")
+        metadataTags = {OptionMetadataTag.DEPRECATED},
+        help = "The Android target CPU. Will not have an effect if --fat_apk_cpu has a value, "
+                + "even a default one. It is recommended to just use --fat_apk_cpu.")
```

Comment:
We should not recommend fat_apk_cpu, as we will be removing it sometime this year in favor of using --android_platforms.
"even a default one" doesn't make sense.  fat_apk_cpu has a default, so that would mean android_cpu does nothing.

The safest correct thing is to just stop after "... --fat_apk_cpu has a value."

Reply:
+1 to this comment.

Reply:
This is fair. I'm going to leave it to @ahumesky to be the decider: what documentation should we have on a flag that will be obsolete in the future?